### DISCUSSION TITLE
Interesting papers

### DISCUSSION THREAD

**Discussion by DominiqueMakowski** [itsdfish/SequentialSamplingModels.jl#57] [General]
Discussion to list relevant papers related to cognitive modelling and SSMs.

- Are non-decision time indices wrong? (Bompas et al., preprint)
  - https://www.biorxiv.org/content/biorxiv/early/2023/02/21/2023.02.20.529290.full.pdf

**Comment by itsdfish** [comment]
This article looks interesting. Good find! Its on my list.

**Comment by DominiqueMakowski** [comment]
Happy new year!

Just mentioning https://www.researchgate.net/publication/387818183_Parameter_identifiability_in_evidence-accumulation_models_The_effect_of_error_rates_on_the_diffusion_decision_model_and_the_linear_ballistic_accumulator

TLDR; LBA might not be appropriate if the amount of "errors" is low (< 5-15%)

**Reply by itsdfish** [reply]
Happy new year!

Thanks for the paper. One hypothesis I have, without having read the paper, is that there is not much information in the survivor function the PDF: fi(t) Sj(t). Because Sj(t) is an integral, it may not provide much information about accumulator j for different values of t.
